spark-bigquery-dsv2/spark-bigquery-dsv2-common/src/main/java/com/google/cloud/spark/bigquery/v2/context/BigQueryIndirectWriterCommitMessageContext.java

```diff
@@ -0,0 +1,30 @@
+/*
```

Comment:
Will changes for the data source writer be created in a new PR?

Reply:
Yes - smaller PRs (as much as 23 files are involved...) are easier to review. The writer will be in a subsequent PR.